t/t4215-log-bloom.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+test_description='git log for a path with bloom filters'
+. ./test-lib.sh
+
+GIT_TEST_COMMIT_GRAPH_BLOOM_FILTERS=0
+
+test_expect_success 'setup the full repo' '
+	mkdir full &&
+	cd "$TRASH_DIRECTORY/full" &&
+	git init &&
+	git config core.commitGraph true &&
+	objdir=".git/objects" &&
+	test_oid_init
+'
+
+test_expect_success 'create 9 commits and repack' '
+	cd "$TRASH_DIRECTORY/full" &&
+	
+	echo file1 >> file1 &&
+	git add . &&
+	test_tick &&
+	git commit -m c1 && 
+	
+	echo file2 >> file2 &&
+	git add . &&
+	test_tick &&
+	git commit -m c2 &&
+	
+	echo file3 >> file3 &&
+	git add . &&
+	test_tick &&
+	git commit -m c3 &&
+	
+	echo file1 >> file1 &&
+	git add . &&
+	test_tick &&
+	git commit -m c4 && 
+	
+	echo file2 >> file2 &&
+	git add . &&
+	test_tick &&
+	git commit -m c5 &&
+	
+	echo file3 >> file3 &&
+	git add . &&
+	test_tick &&
+	git commit -m c6 &&
+	
+	echo file1 >> file1 &&
+	git add . &&
+	test_tick &&
+	git commit -m c7 && 
+	
+	echo file2 >> file2 &&
+	git add . &&
+	test_tick &&
+	git commit -m c8 &&
+	
+	echo file3 >> file3 &&
+	git add . &&
+	test_tick &&
+	git commit -m c9 &&
+	
+	git repack
+'
+
+printf "c7\nc4\nc1" > expect
+
+test_expect_success 'log without bloom filters' '
+	cd "$TRASH_DIRECTORY/full" &&
+	git log --pretty="format:%s"  -- file1 > actual &&
+	test_cmp expect actual
+'
```

Comment:
Instead, focus on comparing the output from `git log` with and without the commit-graph using `-c core.commitGraph=false`.

Reply:
(i.e. drop this part, and I'll comment later about what to do.)